src/app/loginComponents/accounts/external/accounts.component.html

```diff
@@ -1,44 +1,50 @@
+<hr />
 <div *ngFor="let row of accountsInfo">
-  <div class="panel panel-default">
-    <div class="panel-heading">
-      <h4 class="panel-title">{{ row.name }} Account
-        <span *ngIf="row.isLinked">
-          <span class="label label-success">
-            <span class="glyphicon glyphicon-link"></span>
-            <span>Linked</span>
-          </span>
-          <button class="btn btn-default btn-xs" matTooltip="Copy token" type="button" ngxClipboard [cbContent]="row.source | getTokenContent: tokens"
-            (cbOnSuccess)="isCopied1 = true">
-            <span class="glyphicon glyphicon-copy"></span>
-          </button>
-          <button [matTooltip]="row.show ? 'Hide token' : 'Show token'" class="btn btn-secondary btn-xs" (click)="row.show = !row.show">
-            <span *ngIf="row.show" class="glyphicon glyphicon-eye-close"></span>
-            <span *ngIf="!row.show" class="glyphicon glyphicon-eye-open"></span>
-          </button>
-          <span class="form-inline" *ngIf="row.show">
-            <span class="input-group">
-              <!-- TODO: Only have 1 pipe here -->
-              <span class="form-control input-xs">{{row.source | getTokenContent: tokens}}</span>
+  <mat-card class="mb-2">
+    <div>
+      <h4>
+        <mat-chip-list>
+          <mat-chip *ngIf="row.isLinked" color="accent" selected matTooltip="Account is linked" [matTooltipShowDelay]="1000">
```

Comment:
can remove all matTooltipShowDelay unless it's the intention for it to be special